src/style.css

```diff
@@ -1 +1,31 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+  height: 20px;
+}
+
+.stars:last-of-type {
+  height: 16px;
+}
```

Comment:
it looks redundant
```suggestion
```

Reply:
Not fixed 
<img width="833" alt="image" src="https://github.com/user-attachments/assets/d3db6c44-a1e2-431c-85ac-76405da73404">